Doc/library/types.rst

```diff
@@ -409,7 +409,10 @@ Additional Utility Classes and Functions
                return "{}({})".format(type(self).__name__, ", ".join(items))
 
            def __eq__(self, other):
-               return self.__dict__ == other.__dict__
+               return (
+                   isinstance(self, SimpleNamespace) == isinstance(other, SimpleNamespace)
```

Comment:
I think that should be `and`, not `==`. Although it might not make a difference, using `and` would give you short-circuiting. Otherwise, looks good. I think this version is appropriate for the "rough equivalent" documentation.

Reply:
Shouldn't the method return `NotImplemented` if the *other* type mismatches?
Is there a possibility when `self` is not an instance of `SimpleNamespace`?

Reply:
Thank you both for your time and the reviews.

I updated the PR with both suggested changes.